locales/en-US/app.ftl

```diff
@@ -150,6 +150,23 @@ CallTree--bytes-self = Self (bytes)
         The “self” bytes are useful for understanding where memory was actually
         allocated or deallocated in the program.
 
+## Call tree "badges" (icons) with tooltips
+##
+## These inlining badges are displayed in the call tree in front of some
+## functions for native code (C / C++ / Rust). They're a small "inl" icon with
+## a tooltip.
+
+# Variables:
+#   $calledFunction (String) - Name of the function whose call was inlined.
+CallTree--divergent-inlining-badge =
```

Comment:
We don't have a fall back text in this one like the next one. Is this intended?

Reply:
Yes this is intended. `(inlined)` is short and to the point and I couldn't think of a similarly succinct string for "sometimes inlined, or inlined into different functions in different stacks". So I'd rather have screen readers just skip this one entirely. It's rare anyway.